components/Work.vue

```diff
@@ -225,10 +265,16 @@ section.links
   section
     margin: 0
 
-section.tags
-li
+section.tags li
   display inline-block
 
+.multiple-images-horizontal
+  ul
+    display grid
+    max-width: 100%
+    grid-template-columns repeat(3, 1fr)
+  li img
+    width 100%
```

Comment:
Maybe if I do `li, li img` instead of `li`, the inconsistent width issue will be gone?

Reply:
This was due to the user agent's `margin` on the `<figure>` element. Setting it to `0` fixes the iconsistent width issue.